README.md

```diff
@@ -1 +1,230 @@
-# ballerina-aks-deployment
\ No newline at end of file
+[![Build Status](https://travis-ci.org/ballerina-guides/ballerina-aks-deployment.svg?branch=master)](https://travis-ci.org/ballerina-guides/ballerina-aks-deployment)
+
+# Deployment Ballerina with Azure Kubernetes Service
+
+[AKS](https://azure.microsoft.com/en-us/services/kubernetes-service/) provides an easy to use hosted Kubernetes cluster service, which drastically reduces your time to setup your own infrastructure.
+
+> In this guide you will learn about building a Ballerina service and deploying it on Azure Kubernetes Service (AKS).
+
+The following are the sections available in this guide.
+
+- [What you'll build](#what-youll-build)
+- [Prerequisites](#prerequisites)
+- [Implementation](#implementation)
+- [Deployment](#deployment)
+- [Testing](#testing)
+
+## What you’ll build 
+
+In this guide, you will build a simple Ballerina service that generates an UUID each time, and you will deploy this service on AKS. 
+
+## Compatibility
+
+| Ballerina Language Version 
+| -------------------------- 
+| 0.982.0
+
+## Prerequisites
+ 
+- [Ballerina Distribution](https://ballerina.io/learn/getting-started/)
+- A Text Editor or an IDE 
+- [Docker](https://docs.docker.com/engine/installation/)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [Docker Hub Account](https://hub.docker.com/)
+- [Microsoft Azure Account](https://azure.microsoft.com/en-us/free/)
+
+### Optional requirements
+
+- Ballerina IDE plugins ([IntelliJ IDEA](https://plugins.jetbrains.com/plugin/9520-ballerina), [VSCode](https://marketplace.visualstudio.com/items?itemName=WSO2.Ballerina))
+
+## Implementation
+
+As the first step, you can build a Ballerina service that gives an UUID as the output. You can simply create a file `uuid_service.bal` and add the following content to the service code.
+
+```ballerina
```

Comment:
Shall we add some more comments to this code block? Ideally a user should be able to see this code, understand, and customize it.